graphql_compiler/query_pagination/query_parameterizer.py

```diff
@@ -158,12 +162,13 @@ def _add_pagination_filter_at_node(
                     if not _is_new_filter_stronger(
                         operation, new_directive_parameter_value, parameter_value
                     ):
-                        raise AssertionError(
-                            f"Pagination filter {directive_to_add} on "
+                        logger.error(
+                            f"Pagination filter {print_ast(directive_to_add)} on "
                             f"{pagination_field} is not stronger than "
-                            f"an existing filter {directive}. This is "
+                            f"an existing filter {print_ast(directive)}. This is "
                             f"likely a bug in parameter generation. "
-                            f"Query string: {query_string}"
+                            f"Query string: {query_string}, parameters: "
+                            f"{extended_parameters}."
```

Comment:
Can you include the old/new parameter values adjacent to the directives they correspond to?